### DISCUSSION TITLE
Closing old issues

### DISCUSSION THREAD

**Discussion by greglandrum** [rdkit/rdkit#7815] [Development]
As part of the Hackathon at the RDKit UGM this year, a group of people reviewed issues here in github that had not been active in a long time and started closing the ones that had been idle for a long time, looked like they had already been fixed, were questions, or were unlikely to be fixed.

If we closed an issue you created while doing this and you think it should be reopened, feel free to comment in the issue itself (or in this thread) and we will take a look.

By the way: many thanks to the team that put in the time to do this work at the Hackathon! :-)

**Comment by UnixJunkie** [comment]
Some projects have some setup to do this automatically (issues are closed after some elapsed time of inactivity).
I think it is a pretty aggressive strategy, but I understand that some projects don't have the manpower to thieve through lots of github issues.
